src/main/java/frc/robot/subsystems/ball/Ball.java

```diff
@@ -0,0 +1,7 @@
+package frc.robot.subsystems.ball;
+
+public interface Ball {
+    void shoot();
+    void cancelShoot();
```

Comment:
If it's `shoot`/`cancelShoot`, where is the corresponding `cancelUnload`? Could it simply be `cancel`, which stops everything?

Reply:
Couldn't name it cancel as it interferes with RepeatingPooledSubsystem.

Reply:
Okay, so is there some other way it could be named so that it corresponds to cancelling all actions, not only shooting?

Reply:
I could just make it cancelAll, but when I was originally writing it I didn't think of cancelling the unload as well. I'll do that now.

Reply:
Actually, cancelShoot would cancel the unload as well, because it stops both the shooter and the feeder. I just named it incorrectly.

Reply:
Maybe just `stop` or something would be an appropriate name then?

Reply:
Yeah that would work too